roles/openmrs/tasks/main.yml

```diff
@@ -70,6 +70,12 @@
     state: link
     mode: 0777
 
+- set_fact:
```

Comment:
If you had a variable, let's say `openmrs_major_version` that's set to either `1` or `2` then let the `openmrs_version` hold the minor version, you wouldn't have to do this.